src/azure-cli/azure/cli/command_modules/monitor/_help.py

```diff
@@ -748,6 +748,17 @@
 short-summary: Retrieve service diagnostic settings categories.
 """
 
+helps['monitor diagnostic-settings categories list'] = """
+type: command
+short-summary:  Lists the diagnostic settings categories for the specified resource.
+examples:
+  - name: List diagnostic settings categories by using resource ID
+    text: az monitor diagnostic-settings categories list --resource /subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/myRG/providers/microsoft.logic/workflows/myworkflow
```

Comment:
I saw myResource and MyResource. Any convention?

Reply:
In my opinion, it should be myResource. I will double check this PR.